src/sentry/runner/initializer.py

```diff
@@ -97,6 +94,27 @@ def init_plugin(plugin):
             q.durable = False
             settings.CELERY_QUEUES.append(q)
 
+    # TODO: Register only during a `lims upgrade`
+    # Make sure we have a plugin registration here:
+    from clims.models import PluginRegistration
+    from sentry.models import Organization
+    plugin_type = type(plugin)
+    try:
+        plugin_version = plugin.version
+    except AttributeError:
+        plugin_version = "NA"
+
+    try:
+        registered_name = "{}.{}".format(plugin_type.__module__, plugin_type.__name__)
+        # NOTE: Registration currently happens for all organizations:
+        for org in Organization.objects.all():
+            PluginRegistration.objects.get_or_create(
```

Comment:
I think will need a walkthrough, in future,  of how plugins are to be used, and how they relates to organizations. Here all found plugins are registered to all organizations. Is that the intention?

Reply:
Right, for now they are just registered for all organizations, with the possibility to build on top of this in the future. It would probably be preferable not to have the same plugins registered for all plugins, but I think we can live with it like this in 1.0.